app/controllers/application_controller.rb

```diff
@@ -1,4 +1,12 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+  include Errors::ErrorsHandler
+  include Authenticable
```

Comment:
Do you think it will be used widely or just in LoginController? Maybe it's worth to include it only there?

Reply:
skiThis gives the `current_user` method so we should leave it there. 
If mostly we we will have authenticated endpoints we can set before filter here with authenticated user and then in controllers where we don'w want `skip...` ?